### PR TITLE
Added ability to execute the refresh_course_metadata management command via HTTP

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-4204

### CODE REVIEW

**Comment by clintonb** [comment]
FYI @bderusha @peter-fogg this a workaround until we get the automated job running.

File: course_discovery/apps/api/v1/views.py [hunk 1]
```diff
@@ -156,3 +160,47 @@ def list(self, request, *args, **kwargs):
...
+        log_handler = logging.StreamHandler(log)
+        formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+        log_handler.setFormatter(formatter)
+        root_logger.addHandler(log_handler)
+
+        logger.info('Updating course metadata per request of [%s]...', request.user.username)
+
+        kwargs = {}
+        if access_token:
+            kwargs.update({'access_token': access_token})
```
Comment (peter-fogg) [inline course_discovery/apps/api/v1/views.py]: `kwargs = {'access_token': access_token} if access_token else {}`

**Comment by bderusha** [comment]
👍

**Comment by peter-fogg** [comment]
:+1:

File: course_discovery/apps/api/v1/views.py (hunk 1 above)
Comment (clintonb) [inline course_discovery/apps/api/v1/views.py]: Updated.

**Pull request merged**